ioctl/cmd/node/nodedelegate.go

```diff
@@ -42,8 +42,8 @@ const (
 // Multi-language support
 var (
 	delegateCmdUses = map[config.Language]string{
-		config.English: "delegate [-e epoch-num|-n] [-a]",
-		config.Chinese: "delegate [-e epoch数|-n] [-a]",
+		config.English: "delegate [-e epoch-num]",
+		config.Chinese: "delegate [-e epoch数]",
```

Comment:
`-a` cannot be deleted

Reply:
It seems sortAndPrint() print the all delegates by default.

Reply:
By default, it shows top 36. With -a, it shows all